src/Fleet/Fleet.Autorest/docs/Update-AzFleet.md

```diff
@@ -8,38 +8,27 @@ schema: 2.0.0
 # Update-AzFleet
 
 ## SYNOPSIS
-Update a Fleet
+Create a Fleet.
```

Comment:
The verb should be Update. Needs a bug fix for autorest.

Reply:
Can this pr be merged then ?

Reply:
Please hold on this PR until this synopsis is corrected.